pyprotista/parsers/ident/comet_2020_01_4_parser.py

```diff
@@ -1,65 +1,140 @@
 """Engine parser."""
-import multiprocessing as mp
-import sys
-from io import BytesIO
 
 import numpy as np
 import pandas as pd
 import regex as re
-from loguru import logger
-from lxml import etree
-from tqdm import tqdm
+import xml.etree.cElementTree as etree
 
 from pyprotista.parsers.ident_base_parser import IdentBaseParser
 
+element_tag_prefix = "{http://psidev.info/psi/pi/mzIdentML/1.2}"
```

Comment:
namespace should be parsed from file:
https://github.com/computational-ms/pyProtista/blob/ec876bbd40c40568583df782a643862bb5957e0d/tests/data/BSA1_comet_2020_01_4.mzid#L14

Reply:
But on the other hand, another mzIdentML format could potentailly break the parser anyway, so alternatively we could check if the version is the hardcoded one here and else raise an exception/warning

Reply:
maybe just check once in the beginning, raise warning if needed, and for all equality assertions use .endswith instead

Reply:
now uses tag.endswith.
i implemented a warning at an early entry to check the tag